packages/cli/src/services/frontend.service.ts

```diff
@@ -204,6 +204,10 @@ export class FrontendService {
 				pruneTime: -1,
 				licensePruneTime: -1,
 			},
+			n8nPath: {
+				path: config.getEnv('path'),
```

Comment:
I think this already gets set as `window.BASE_PATH`. you can find it also as `baseUrl` in root store already. so we don't need to add these settings

Reply:
Ah, good point. Updated